src/renderer.cpp

```diff
@@ -596,25 +626,50 @@ void Renderer::shutdown()
 
     bgfx::shutdown();
 }
+void Renderer::clearShadowMap()

```

Comment:
See the relative comment I left about this topic. We should avoid reserving view IDs.

Reply:
See the comment above for the reasons I think reserving view id 0 for shadowmap rendering is reasonable.

Reply:
Replied, thanks